src/components/UI/VersionPicker/VersionPicker.tsx

```diff
@@ -101,25 +101,47 @@ const Body = styled.div`
   }
 `;
 
+interface IVersion {
+  // The version
+  version: string;
+
+  // Whether or not this version is a test version.
+  test: boolean;
+}
+
+interface IVersionPickerProps {
+  // An array of versions to pick from.
+  versions: IVersion[];
+
+  // The currently selected version, will be highlighted in bold in the list.
+  selectedVersion: string;
+
+  // A callback function that gets called when a version is selected by the user.
+  onChange?: (newVersion?: string) => void;
+}
+
```

Comment:
Do you follow an y rules when deciding to use a `type` or an `interface`? Interfaces are more powerful, but we could use the convention to use `type`s whenever something is not meant to be reused/extended. WDYT?

Reply:
> Interfaces over types for Map-like data structures (basically plain JS objects) -> interfaces can be implemented, extended and merged

or taken from the TypeScript documentation:
> Because an ideal property of software is being open to extension, you should always use an interface over a type alias if possible.

I'd say we should be consistent and always use interfaces for objects with a map-like structure, example:
```javascript
const someObj = {
  someKey: 'someValue',
}
```

This way there would not be any arguing in PRs whether to use types or interfaces. 
Also, theoretically speaking, objects in JS are all the same type, so it makes sense to define them with interfaces, since they just respect a certain blueprint.

There are special cases where only `type` works (such as mapped types):
```typescript
// Built-in Record type
type Record<K extends keyof any, T> = {
    [P in K]: T;
}
```

There's a small doc about this in the TypeScript documentation:
https://www.typescriptlang.org/docs/handbook/advanced-types.html#interfaces-vs-type-aliases